include/SFML/System/Span.inl

```diff
@@ -0,0 +1,351 @@
+
+// TODO license text for a modified work
+
+/*
+This is an implementation of C++20's std::span
+http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/n4820.pdf
+*/
+
+//          Copyright Tristan Brindle 2018.
+// Distributed under the Boost Software License, Version 1.0.
+//    (See accompanying file ../../LICENSE_1_0.txt or copy at
+//          https://www.boost.org/LICENSE_1_0.txt)
```

Comment:
Is the Boost license really compatible with SFML's zlib license?

Reply:
These are the relevant parts:

> **Permission is hereby granted**, free of charge, to any person or organization
obtaining a copy of the software and accompanying documentation covered by
this license (the "Software") to use, reproduce, display, distribute,
execute, and transmit the Software, and **to prepare derivative works of the
Software**, and to permit third-parties to whom the Software is furnished to
do so, all **subject to the following:**
> 
> **The copyright notices in the Software and this entire statement**, including
the above license grant, this restriction and the following disclaimer,
**must be included in all copies of the Software**, in whole or in part, and
all derivative works of the Software, unless such copies or derivative
works are solely in the form of machine-executable object code generated by
a source language processor.
>
> [no warranty etc]

So, looks like you can do anything, but need to ship the license statement somewhere. I don't know if linking to it counts as "shipping" in legal terms.